EnglishFilter/sections/specific.txt

```diff
@@ -1294,6 +1295,9 @@ businessinsider.in##.ad-brdr-top
 unicode-table.com##.content-inside-banner
 unicode-table.com##.main__top-updated-ad
 nairaland.com##.vertipics
+thebetterindia.com###tbh-offer-popup-wrapper
+thebetterindia.com##.tbi_content
+thebetterindia.com##div[class^="adp-wrapper"]
```

Comment:
this rule will block a player in articles. Added `thebetterindia.com##body .adp-wrapper:not([data-id="5"])`